### DISCUSSION TITLE
Unknown Error Occurred.  How to clear?

### DISCUSSION THREAD

**Discussion by biggen1684** [jgyates/genmon#1156] [Questions] [answered]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open issues](https://github.com/jgyates/genmon/issues),[closed issues](https://github.com/jgyates/genmon/issues?q=is%3Aissue+is%3Aclosed),[open discussion](https://github.com/jgyates/genmon/discussions), [closed discussion](https://github.com/jgyates/genmon/discussions/categories/questions?discussions_q=is%3Aclosed) - which may answer my question or that I could contribute my information to.
- [X] I have read the [Known Issues Wiki Page](https://github.com/jgyates/genmon/wiki/Appendix-D-Known-Issues) and my question / issue is not listed.
- [X] I have provided the version of the software and the operating system in my post.
- [X] If I have an issue with genmon not communicating with the controller, I have read and followed the [serial troubleshooting guide](https://github.com/jgyates/genmon/wiki/3.6---Serial-Troubleshooting).

### Version

Version V1.19.03

### Have you submitted your log files from the About page? Note that outbound email must be working for logs to be submitted.

I have submitted logs

### What is/are your question(s)?

Seeing an "Unknown Error Occurred" in the bottom of the genmon dashboard.  Wants me to turn on "Auto Feedback".  I went ahead and submitted the generator registers and logs manually instead (12:07pm and 12:12pm CST they were emailed).

I'm not sure what/when that error message is from.  Any errors before 7/26 should probably be ignored since I finally got all the pieces installed out there at the generator yesterday after several test setups.  Had to run ethernet out there as wifi was just too flaky.

Does that error message indicate something happen prior to 7/26 or after 7/26?  And is there any way to clear it?

**Comment by jgyates** [comment, marked as answer]
Genmon will send an email to the developer (me) if an unknown value is read from the controller, but only if "Auto Feedback" is enabled. The Evolution 2.0 controller will report invalid data at times (see the [Known Issues Wiki entry](https://github.com/jgyates/genmon/wiki/Appendix-D-Known-Issues#11-evolution-20-artifact) for more details).

According to your logs it looks like this is what occurred. 

You can:

Enable Auto Feedback and it will send the snapshot of registers that were saved and send them in the future if this occurs
or you can delete the file /etc/genmon/feedback.json and the message will go away.

**Reply by biggen1684** [reply]
Thanks!  I'll enable Auto Feedback.

**Comment by Jkh62** [comment]
Was not aware of the auto -feedback feature. 
I switched it on.
